script.js

```diff
@@ -19,3 +19,173 @@ const initApp = () => {
+    sourceLink: 'https://example.com/source1',
+  },
+
+  {
+    name: "Most-Post Storie's <br/> Gain+Glory",
+    description: 'Project 2 description',
+    image: { mobile: './images/portfolio-projects.png', desktop: './images/desk-porto.png' },
+    technologies: ['Ruby on Rails', 'CSS', 'JavaScript', 'HTML'],
+    liveLink: 'https://example.com/live2',
+    sourceLink: 'https://example.com/source2',
+  },
+
+  {
+    name: "Most-Post Storie's <br/> Gain+Glory",
+    description: 'Project 3 description',
+    image: { mobile: './images/portfolio-projects.png.png', desktop: './images/desk-porto.png' },
+    technologies: ['Ruby on Rails', 'CSS', 'JavaScript', 'HTML'],
+    liveLink: 'https://example.com/live2',
+    sourceLink: 'https://example.com/source2',
+  },
+
+  {
+    name: "Most-Post Storie's <br/> Gain+Glory",
+    description: 'Project 4 description',
+    image: { mobile: './images/portfolio-projects.png', desktop: './images/desk-porto.png' },
+    technologies: ['Ruby on Rails', 'CSS', 'JavaScript', 'HTML'],
+    liveLink: 'https://example.com/live2',
+    sourceLink: 'https://example.com/source2',
+  },
+
+  {
+    name: "Most-Post Storie's <br/> Gain+Glory",
+    description: 'Project 5 description',
+    image: { mobile: './images/portfolio-projects.png', desktop: './images/desk-porto.png' },
+    technologies: ['Ruby on Rails', 'CSS', 'JavaScript', 'HTML'],
+    liveLink: 'https://example.com/live2',
+    sourceLink: 'https://example.com/source2',
+  },
+
+  {
+    name: "Most-Post Storie's <br/> Gain+Glory",
+    description: 'Project 6 description',
+    image: { mobile: './images/portfolio-projects.png', desktop: './images/desk-porto.png' },
+    technologies: ['Ruby on Rails', 'CSS', 'JavaScript', 'HTML'],
+    liveLink: 'https://example.com/live2',
+    sourceLink: 'https://example.com/source2',
+  },
+];
+
+const projectsContainer = document.querySelector('.cards');
+
+projects.forEach((project) => {
+  const projectElement = document.createElement('div');
+  projectElement.classList.add('card');
+
+  const imageElement = document.createElement('div');
+  imageElement.classList.add('card-img');
+  projectElement.appendChild(imageElement);
+
+  const contentElement = document.createElement('div');
+  contentElement.classList.add('info');
+  projectElement.appendChild(contentElement);
+
+  const titleElement = document.createElement('h3');
+  titleElement.classList.add('project-title');
+  titleElement.innerHTML = project.name;
+  contentElement.appendChild(titleElement);
+
+  const langListElement = document.createElement('ul');
+  langListElement.classList.add('languages');
+  contentElement.appendChild(langListElement);
+
+  project.technologies.forEach((technology) => {
+    const langItemElement = document.createElement('li');
+    langItemElement.classList.add('language');
+    langItemElement.textContent = technology;
+    langListElement.appendChild(langItemElement);
+  });
```

Comment:
- [ ] Nice one dynamically adding the popups information, kindly fix the following design to attain a pixel-perfect design, as you'll soon be doing in your job 💯 

- Your design
<img width="944" alt="image" src="https://github.com/AlyAtalla/AlyAtalla.github.io/assets/41163571/048a2676-dbc8-42a5-b481-0b19da726989">

- Figma design
<img width="349" alt="image" src="https://github.com/AlyAtalla/AlyAtalla.github.io/assets/41163571/66b6e921-d64f-4f42-85dc-1dd474b3e758">

**Kindly fix the following**
- The font family doesn't match the design, you'll see the properties at the right-hand frame of the interface here.
<img width="636" alt="image" src="https://github.com/AlyAtalla/AlyAtalla.github.io/assets/41163571/2177b93c-cc5b-46c4-b67e-6c5f357733b0">.
- The image should match that of the design.
- The image sub-heading in relation to the buttons.
- The language lists.
- The ultimate design of the popup is not in line with that of the Figma design, Kindly fix this, please.